tree/dataframe/inc/ROOT/RDFInterface.hxx

```diff
@@ -428,12 +438,12 @@ public:
 
    ////////////////////////////////////////////////////////////////////////////
    /// \brief Save selected columns to disk, in a new TTree `treename` in file `filename`.
-   /// \tparam BranchTypes variadic list of branch/column types
-   /// \param[in] treename The name of the output TTree
-   /// \param[in] filename The name of the output TFile
-   /// \param[in] columnList The list of names of the columns/branches to be written
-   /// \param[in] options RSnapshotOptions struct with extra options to pass to TFile and TTree
-   /// \return a `RDataFrame` that uses the snapshotted tree as dataset
+   /// \tparam ColumnTypes variadic list of branch/column types.
```

Comment:
This is a good idea. I wonder if this would not be redundant with the signature for some of the methods. For methods like Min, absolutely. For Snapshot, if the method is template, the arguments are not optional. Maybe we can remind the user that it exists a sister method for which the template arguments are not necessary?